<ui version="4.0">
 <class>SelectPlaceDlg</class>
 <widget class="QWidget" name="SelectPlaceDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QRadioButton" name="rbLocalPlace">
     <property name="text">
      <string>Local Place</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>19</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="KUrlRequester" name="urLocalPlaceUrl"/>
     </item>
    </layout>
   </item>
   
   <item>
    <widget class="QRadioButton" name="rbLocalFile">
     <property name="text">
      <string>Local File</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>19</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="KUrlRequester" name="urLocalFileUrl"/>
     </item>
    </layout>
   </item>
   
   <item>
    <widget class="QRadioButton" name="rbRemotePlace">
     <property name="text">
      <string>Remote URL</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>16</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="lbRemoteUrl">
           <property name="text">
            <string>URL:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leRemoteUrl">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="gbRemoteHelp">
         <property name="title">
          <string>Help</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="lbProtocol">
            <property name="text">
             <string>Protocol:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cbProtocol">
            <property name="editable">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string>http</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>https</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ftp</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>sftp</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>smb</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QCheckBox" name="cbAuthentification">
            <property name="text">
             <string>Use Authentication</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="lbUser">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>User:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="leUser">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="lbPassword">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Pass:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="lePass">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lbHost">
            <property name="text">
             <string>Host:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="leHost"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lbPath">
            <property name="text">
             <string>Path:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lePath"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>rbLocalPlace</tabstop>
  <tabstop>rbLocalFile</tabstop>
  <tabstop>rbRemotePlace</tabstop>
  <tabstop>leRemoteUrl</tabstop>
  <tabstop>gbRemoteHelp</tabstop>
  <tabstop>cbProtocol</tabstop>
  <tabstop>leHost</tabstop>
  <tabstop>lePath</tabstop>
  <tabstop>cbAuthentification</tabstop>
  <tabstop>leUser</tabstop>
  <tabstop>lePass</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cbAuthentification</sender>
   <signal>toggled(bool)</signal>
   <receiver>leUser</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbAuthentification</sender>
   <signal>toggled(bool)</signal>
   <receiver>lePass</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbAuthentification</sender>
   <signal>toggled(bool)</signal>
   <receiver>lbUser</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbAuthentification</sender>
   <signal>toggled(bool)</signal>
   <receiver>lbPassword</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gbRemoteHelp</sender>
   <signal>toggled(bool)</signal>
   <receiver>lbProtocol</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gbRemoteHelp</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbProtocol</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gbRemoteHelp</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbAuthentification</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gbRemoteHelp</sender>
   <signal>toggled(bool)</signal>
   <receiver>leUser</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gbRemoteHelp</sender>
   <signal>toggled(bool)</signal>
   <receiver>lePass</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gbRemoteHelp</sender>
   <signal>toggled(bool)</signal>
   <receiver>lbUser</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gbRemoteHelp</sender>
   <signal>toggled(bool)</signal>
   <receiver>lbPassword</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>339</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbRemotePlace</sender>
   <signal>toggled(bool)</signal>
   <receiver>lbRemoteUrl</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>53</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbRemotePlace</sender>
   <signal>toggled(bool)</signal>
   <receiver>leRemoteUrl</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbRemotePlace</sender>
   <signal>toggled(bool)</signal>
   <receiver>gbRemoteHelp</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gbRemoteHelp</sender>
   <signal>toggled(bool)</signal>
   <receiver>leHost</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gbRemoteHelp</sender>
   <signal>toggled(bool)</signal>
   <receiver>lbHost</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gbRemoteHelp</sender>
   <signal>toggled(bool)</signal>
   <receiver>lePath</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gbRemoteHelp</sender>
   <signal>toggled(bool)</signal>
   <receiver>lbPath</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbLocalPlace</sender>
   <signal>toggled(bool)</signal>
   <receiver>urLocalPlaceUrl</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>17</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  
  <connection>
   <sender>rbLocalFile</sender>
   <signal>toggled(bool)</signal>
   <receiver>urLocalFileUrl</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>17</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
